src/HDF5.jl

```diff
@@ -41,6 +41,7 @@ include("datafile.jl")
 
 function h5_get_libversion()
```

Comment:
just move this to the end of the file

Reply:
The `read_const` calls are actually relying on this having already been run to set on line 50 where `const libversion = h5_get_libversion()`. That's part of the initialization mess that needs to get sorted out.

(The alternative is to move this lower in the file and instead just call `h5_open()` directly before defining and using `read_const()`.)